nvmeadm/src/nvmf_discovery.rs

```diff
@@ -39,7 +44,12 @@ static HOST_ID: once_cell::sync::Lazy<String> = once_cell::sync::Lazy::new(|| {
             }
         }
     }
-    host_id
+    let nqn_prefix = "nqn.2014-08.org.nvmexpress:uuid";
```

Comment:
Because newer kernels no longer allow multiple combinations of nqn to id
This allows us to use nvme connect whilst testing, without having to specify hostid.
For "production control-plane" we specify the hostnqn rather than take default.